recipes/json-c/meta.yaml

```diff
@@ -0,0 +1,35 @@
+package:
+    name: json-c
+    version: 0.12
+
+source:
+    fn: json-c-0.12.tar.gz
+    url: https://s3.amazonaws.com/json-c_releases/releases/json-c-0.12.tar.gz
+    md5: 3ca4bbb881dfc4017e8021b5e0a8c491
+
+build:
+    number: 0
+    skip: True  # [win]
+
+requirements:
+    build:
+        - autoconf
+        - automake
+        - libtool
+    run:
+        - libgcc
```

Comment:
Issue like [this](https://github.com/ioos/conda-recipes/issues/723) are solved by adding `libgcc` from the default channel. In this case I think we are OK without it though because I used `conda-inspect` and I there are no links to gomp.